src/core/js/startController.js

```diff
@@ -14,16 +14,7 @@ define([
             if (!this.isEnabled()) return;
 
             var hash = this.getStartHash();
```

Comment:
the local variable `hash` is now redundant, might as well delete this line and change the following line to:
```js
window.history.replaceState('', '', this.getStartHash());
```